ajax/libs/tiny-slider/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "tiny-slider",
+  "filename": "tiny-slider.js",
+  "version": "2.1.2",
+  "description": "Vanilla javascript slider for all purposes, inspired by Owl Carousel.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ganlanyuan/tiny-slider.git"
+  },
+  "keywords": [
+    "vanilla javascript",
+    "tiny slider",
+    "slider"
+  ],
+  "author": "ganlanyuan <ganlanyuan@gmail.com>",
+  "license": "MIT",
+  "npmName": "tiny-slider",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "homepage": "http://ganlanyuan.github.io/tiny-slider/"
```

Comment:
Let's use `https`
Using https is better because it's more security for browsing. To prevent the possibility the users been attacked during the redirection.
Thank you.